src/js/components/DateInput/utils.js

```diff
@@ -125,6 +125,30 @@ const pullDigits = (text, index) => {
   return text.slice(index, end);
 };
 
+export const validateDateBounds = ({ dateBounds, selectedDate }) => {
```

Comment:
```suggestion
export const validateDateBounds = (dateBounds, selectedDate) => {
```
This is related to my comment above about just passing as function parameters which is a little more aligned to other functions in this file.

Reply:
done